### PR TITLE
expand contributing docs

### PR DESCRIPTION
**Pull request by BenTheElder**
this is a first pass at better covering:
- how to get involved
- what tools to install
- project scope and priorities
- organizing the documentation

I also added some basic `<hr>` styling since we're now using those.

We will need to iterate on this more. I'd like to expand details of all of this soon, but it was already quite a lot so I'm putting up the first pass.

/cc @neolit123 @munnerz

### CODE REVIEW

**Comment by BenTheElder** [comment]
https://deploy-preview-301--k8s-kind.netlify.com/

**Comment by BenTheElder** [comment]
other than iterating on these I plan to add a F.A.Q. next since we have a number of those.

File: site/content/docs/contributing/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+```
+go version
+```
+This documentation is written using Go version 1.11+.
+
+#### Install or upgrade Docker
+If you haven't already, install the 
+[Docker software using the instructions for your operating system][docker].
+If you have an existing installation, check your version and make sure you have
+the latest Docker.
```
Comment (neolit123) [inline site/content/docs/contributing/getting-started.md]: [1]

File: site/content/docs/contributing/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+To check if `docker` is has been installed:
+```
+docker --version
+```
+This documentation is written using Docker version 18.09.0.
```
Comment (neolit123) [inline site/content/docs/contributing/getting-started.md]: [1]
recommend Docker version related to runc - e.g. 18.09.2?

File: site/content/docs/contributing/project-scope.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+**Stakeholders**: 
+
+- SIG Testing
+- SIG Cluster-Lifecycle
+  - the kubeadm subproject
+- Possibly SIG Release (mainly to provide easy access to alpha and beta tags)
+
+**Covered Work**:
+
+- limited workloads / e2e testing
```
Comment (neolit123) [inline site/content/docs/contributing/project-scope.md]: minor nit: uppercase bullet entries bellow, unless a lowercase project name - e.g. `cluster-api-..`?

Comment (BenTheElder) [reply site/content/docs/contributing/project-scope.md]: done

File: site/content/docs/contributing/getting-started.md (hunk 2 above)
Comment (BenTheElder) [reply site/content/docs/contributing/getting-started.md]: thanks! done

File: site/content/docs/contributing/getting-started.md (hunk 1 above)
Comment (BenTheElder) [reply site/content/docs/contributing/getting-started.md]: on the host the latest docker should be fine I think?

Comment (neolit123) [reply site/content/docs/contributing/getting-started.md]: it's fine.

**Comment by munnerz** [comment]
I think there's a few extra clarifications we can add to the project scope section, but I'll try and organise my thoughts and put a PR together in the morning 😄

/lgtm
/hold cancel

**Pull request merged**